api/move_chunk.md

```diff
@@ -1,34 +1,33 @@
-## move_chunk() <tag type="community">Community</tag> 
+## move_chunk() <tag type="community">Community</tag>
```

Comment:
This documentation applies to the existing move_chunk functionality and NOT the experimental one. So this should not be changed at all.